frontend/src/components/Navbar/index.js

```diff
@@ -1,15 +1,43 @@
-import React, { Component } from 'react'
```

Comment:
@krish7777  should we have Navbar as a component?.. I have removed it for now let me know your views!

Reply:
It is now a functional component instead of state component